examples/eval.sh

```diff
@@ -0,0 +1,21 @@
+# We first try to find a output of a prevouis training run
+
+OUTDIR=$(find outputs -type d -mindepth 2 | head -n 1)
+echo $OUTDIR
+
+# And evaluate the model on the training dataset
+
+metatensor-models eval --model=$OUTDIR/model_final.pt \
+                       --structure=qm9_reduced_100.xyz \
+                       --output=output.xyz
+
+# The evaluation command predicts the property the model was trained against; here "U0".
+# The predictions together with the structures have been written in a file named
+# ``output.xyz`` in the current directory. The written file starts with the following
+# lines
+
+head -n 20 output.xyz
+
+# All command line flags of the eval sub-command can be listed via
+
+metatensor-models eval --help
```

Comment:
I feel like this is too advanced as the first example for a first-time user who is potentially not used to programming. To be honest, I have no clue what's going on here

Reply:
I'm not sure what a good solution would be...
head is ok-ish?
But I don't like the find command